code/modules/paperwork/records/info/he.dm

```diff
@@ -3,8 +3,9 @@
 	abno_type = /mob/living/simple_animal/hostile/abnormality/happyteddybear
 	abno_code = "T-04-06"
 	abno_info = list(
-		"An employee died of asphyxiation after working with Happy Teddy Bear multiple times in a row.",
-		"An employee was found dead in Happy Teddy Bear’s arms with a broken neck. It is believed that visiting Happy Teddy Bear’s Containment Unit several times in a row is what causes these unfortunate incidents. Please refer to the Old Letter document for more information.")
+		"An employee died of asphyxiation after working with Happy Teddy Bear multiple times in a row unless the employee has Fortitude Level 3 or higher.",
+		"An employee was found dead in Happy Teddy Bear’s arms with a broken neck. It is believed that visiting Happy Teddy Bear’s Containment Unit several times in a row is what causes these unfortunate incidents. Please refer to the Old Letter document for more information.",
+		"If the employee has Fortitude level 3 or higher, the employee will eventually break free if they stay conscuious")
```

Comment:
if you could, whenever you are changing wrongly formatted lists it'd be great if you fixed them up whilst at it.
```suggestion
		"If the employee has Fortitude level 3 or higher, the employee will eventually break free if they stay conscuious",
	)
```